DB Setup/data_insertion.sql

```diff
@@ -0,0 +1,83 @@
+
+\c va_trial
+
+insert into SessionKeys values
+	(0, 00001, '6:00AM', '6:30AM'),
```

Comment:
These need to be date+times so that there isn't strange behavior as you move to new days

Reply:
done